<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CamCalibGUI</class>
 <widget class="QWidget" name="CamCalibGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="mainGridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="leftwidget" native="true">
     <layout class="QVBoxLayout" name="left_verticalLayout">
      <item>
       <widget class="QGroupBox" name="tuneableParamBox">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="title">
         <string>Tuneable Parameters : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <layout class="QGridLayout" name="boxGridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="focal_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
             <strikeout>false</strikeout>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>f : </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="focal_val">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSlider" name="focal_slider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="alpha_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
             <strikeout>false</strikeout>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>Alpha : </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="alpha_val">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSlider" name="alpha_slider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="beta_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
             <strikeout>false</strikeout>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>Beta :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="beta_val">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSlider" name="beta_slider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="gamma_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
             <strikeout>false</strikeout>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>Gamma : </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="gamma_val">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QSlider" name="gamma_slider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="distance_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
             <strikeout>false</strikeout>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>Distance :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="distance_val">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QSlider" name="distance_slider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
